src/main/java/uk/gov/ons/census/casesvc/config/QueueSetterUpper.java

```diff
@@ -42,6 +42,9 @@
   @Value("${queueconfig.refusal-routing-key}")
   private String caseProcessorRefusalRoutingKeyCase;
 
+  @Value("${queueconfig.fulfilment-routing-key}")
+  private String caseProcessorFulfilmentRoutingKeyCase;
```

Comment:
Tiny (picky!) comment but is it worth removing `caseProcessor` prefix as it lives in this service anyway? I guess the refusal bits are configured in the same way though 🤔